reduction/Imaging_instructions.txt

```diff
@@ -72,3 +72,40 @@ of the important scripts (i.e., this directory:
 https://github.com/ALMA-IMF/reduction/tree/master/reduction).
 
 When you create mask region files, they belong in $ALMAIMF_ROOTDIR/clean_regions/.
+
+
+Additional tricks of cleaning in selfcal
+=============================================
+Be conservative in making the first image before the first iteration of the selfcal.
+That is, clean relatively relatively shallow probably by setting the threshold to be
+4 times rms or a bit greater. Otherwise, some artefacts may be introduced into the final
+image. In the later iterations (e.g., from iter2, 3, ...,) of selfcal, the threshold can be
+constrained down to 2 times rms.
```

Comment:
I don't quite agree with this.  You may need to be quite aggressive in terms of the `tclean` threshold, but you should be somewhat conservative in terms of where you place the mask (or at what level you select the mask threshold).  A failure to include enough flux in the first-pass clean (i.e., by specifying a too-high threshold) can result in bad self-calibration solutions.

Reply:
Got it and learned. In my practice on G353.41, the 4rms as the threshold in the first-pass clean seems work well enough. This could because it is still not too high to cause a hidden trouble in solves of later selfcalibrations. Anyway, now I agree with you that as a general rule the clean could be treated more aggressively (e.g., 2 rms instead of 4rms as the threshold) only if the mask regions are reliable.